lib/ts/recipe/emailverification/recipeImplementation.ts

```diff
@@ -100,16 +94,16 @@ export default function getRecipeImplementation(recipeId: string, appInfo: Norma
                     options,
                     userContext,
                 }),
-                config.postAPIHook
+                Querier.preparePostAPIHook<PreAPIHookAction>({
```

Comment:
Why do we need to give `PreAPIHookAction` in the `preparePostAPIHook`? Can't it infer from the config that's provided to it as the first param? - Discuss with me

Reply:
Youre right, it does complain about typing based on the config. Removed the `<PreAPIHookAction>`